frame/support/test/tests/pallet.rs

```diff
@@ -208,8 +208,7 @@ pub mod pallet {
 
 		/// Doc comment put in metadata
 		#[pallet::weight(1)]
-		#[frame_support::transactional]
```

Comment:
Will you already remove this macro or keep it as NO-OP for compatibility?

Reply:
i was originally thinking about deprecating it, but i didnt see a reason to remove it. It can still be used on top of other functions which are not extrinsics.

https://github.com/paritytech/substrate/pull/11431/commits/5dc88cd720337d8fc2cca64474bf0f230648dcc8

But yeah, in these test contexts, it is not needed anymore

Reply:
Okay I see.  
So then as second step we can remove `#[transactional]` from all extrinsics where we already use it.

Reply:
Yes, there are only some left in the new nomination-pools repo